mbedtls-sys/build/build.rs

```diff
@@ -103,6 +103,22 @@ impl BuildConfig {
             cflags.push("-fno-stack-protector".into());
         }
 
+        // Determine the sysroot for this compiler so that bindgen
+        // uses the correct headers
+        let compiler = cc::Build::new().get_compiler();
+        let output = compiler.to_command().args(&["--print-sysroot"]).output();
```

Comment:
Please run this only if `compiler.is_like_gnu()`

Reply:
Edited in ce2864a.